.idea/libraries/sbt__com_google_protobuf_protobuf_java_3_5_1_jar.xml

```diff
@@ -0,0 +1,11 @@
+<component name="libraryTable">
```

Comment:
Actually I looked at this and I think we should ignore these files (gitignore) instead: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems